### ISSUE TITLE
Roam application freezes when trying to sync photos taken with default resolution of Panasonic FZ-G1. Roam apparently selects highest resolution so photos end up being 6GB in size which causes issues with the sync process.

### ISSUE THREAD

**Issue by PaulGDev** [roam-qgis/Roam#454]

**Comment by PaulGDev** [comment]
Apparently we can put this in the Roam config file but other people have tried this with the Panasonic mk5 with no success.
camera_res: 1920, 1080

**Comment by NathanW2** [comment]
Hey, with the new Roam 3 release this might work better.  Which version of
Roam have you tried that setting with?   The new version will use the Qt5
API which has much much much better camera support and allows me to avoid
all the ugly hacks I had before.

**Comment by PaulGDev** [comment]
Hi

We are using Roam 2.7.1 running on QGIS API 2.18.22 with ECW Support

**Comment by PaulGDev** [comment]
I tried setting camera_res: 1920, 1080 but Roam says Camera failed to start or words to that effect.

**Comment by NathanW2** [comment]
Yeah it won't work very well. Best to grab the latest V3 release and test
that. Note: V3 is still a beta and you need to make backups of your
projects and don't mix them with V2 project at this point.
